relay-server/src/statsd.rs

```diff
@@ -19,20 +19,6 @@ impl GaugeMetric for RelayGauges {
     }
 }
 
-/// Set metrics used by Relay
-pub enum RelaySets {
-    /// Represents the number of active projects in the current slice of time
-    UniqueProjects,
```

Comment:
This metric has been removed since it was unused and not very useful. There's another metric that tracks the number of entries in the project cache.